index.html

```diff
@@ -1,120 +1,138 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
-  </head>
-
-  <body>
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-      <header id="header">
-        <img  src='https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png' id="header-img">
-        <nav id="nav-link">
-          <ul>
-            <li><a>Features</a> </li>
-            <li><a>How It Works</a> </li>
-            <li><a>Pricing</a> </li>
-      
-          </ul>
-        </nav>
-      </header>
-      <h2>Handcrafted, home-made masterpieces</h2>
-      <form action="" id='form'>
-        <input type="search" name="search" id="search" placeholder="Enter your email address">
-        <button class='btn'> <span class="bolder"> GET STARTED</span> </button>
-      </form>
- 
-      
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <header id="header">
+    <img src='https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png' id="header-img"
+      alt="img1">
+    <nav id="nav-link">
+      <ul>
+        <li><a>Features</a> </li>
+        <li><a>How It Works</a> </li>
+        <li><a>Pricing</a> </li>
+
+      </ul>
+    </nav>
+  </header>
+  <h2>Handcrafted, home-made masterpieces</h2>
+  <form action="#" id='form'>
+    <input type="search" name="search" id="search" placeholder="Enter your email address">
+    <button class='btn'> <span class="bolder"> GET STARTED</span> </button>
```

Comment:
The button should have a submit `type=submit`.

Reply:
- [ ] Kindly provide an explanation here as well 🙏 

_See my comment above for more info._

Reply:
You did a great job, but here button should have a submit `type=submit` because when a user presses the button it will submit the request.